src/pkg/util/fs/layout/layer/overlay/overlay.go

```diff
@@ -127,7 +128,8 @@ func (o *Overlay) createLayer(rootFsPath string, system *mount.System) error {
 			}
 			if point.Type == "" {
 				if err := syscall.Stat(point.Source, st); os.IsNotExist(err) {
-					return fmt.Errorf("stat failed for %s: %s", point.Source, err)
+					sylog.Warningf("skipping mount of %s: %s", point.Source, err)
+					continue
```

Comment:
Should we still check `} else if err != nil {` to error out if there's some other form of error that's not `os.IsNotExist()`?